cli/user.go

```diff
@@ -46,13 +42,12 @@ func runGetUser(cmd *cobra.Command, args []string) {
 		cmd.Usage()
 		return
 	}
-	admin := client.NewAdminClient(&context.Context, context.Addr, client.User)
-	body, err := admin.GetYAML(args[0])
+	db := makeSQLClient()
+	err := processOneLine(db, fmt.Sprintf(`SELECT * FROM system.users WHERE username='%s'`, args[0]))
```

Comment:
yup. the cli/sql needs to be reworked quite a bit. For now, I went with expediency to port the user stuff over.